ports/servoshell/headed_window.rs

```diff
@@ -522,33 +517,32 @@ impl WindowPortsMethods for Window {
     }
 }
 
+fn winit_size_to_euclid_size<T>(size: PhysicalSize<T>) -> Size2D<T, DevicePixel> {
+    Size2D::new(size.width, size.height)
+}
+
+fn winit_position_to_euclid_point<T>(position: PhysicalPosition<T>) -> Point2D<T, DevicePixel> {
+    Point2D::new(position.x, position.y)
+}
+
 impl WindowMethods for Window {
     fn get_coordinates(&self) -> EmbedderCoordinates {
-        // Needed to convince the type system that winit's physical pixels
-        // are actually device pixels.
-        let dpr: Scale<f32, DeviceIndependentPixel, DevicePixel> = Scale::new(1.0);
-        let PhysicalSize { width, height } = self.winit_window.outer_size();
-        let PhysicalPosition { x, y } = self
-            .winit_window
-            .outer_position()
-            .unwrap_or(PhysicalPosition::new(0, 0));
-        let win_size = (Size2D::new(width as f32, height as f32) * dpr).to_i32();
-        let win_origin = (Point2D::new(x as f32, y as f32) * dpr).to_i32();
-        let screen = (self.screen_size.to_f32() * dpr).to_i32();
-
-        let PhysicalSize { width, height } = self.winit_window.inner_size();
+        let win_size = winit_size_to_euclid_size(self.winit_window.outer_size()).to_i32();
+        let win_origin = self.winit_window.outer_position().unwrap_or_default();
+        let win_origin = winit_position_to_euclid_point(win_origin).to_i32();
```

Comment:
```suggestion
        let window_size = winit_size_to_euclid_size(self.winit_window.outer_size()).to_i32();
        let window_origin = self.winit_window.outer_position().unwrap_or_default();
        let window_origin = winit_position_to_euclid_point(win_origin).to_i32();
```

I know this `win_` predates your change, but maybe it's a good moment to clean this up.

Reply:
done in 8ba5fbb6da2e6f1d92b2459e33beb6a78a8e453c